SUMMARY.md

```diff
@@ -0,0 +1,37 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > The ternary operator is used as a shorthand way of doing `if/else` statements. In this lesson, you will learn how to use the ternary operator to perform shorthand comparisons in PHP.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to define the data type that is passed in a function declaration.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create, retrieve, modify, and delete cookies using the `setCookie()` function.
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to use session functions to start, get, modify, and destroy a session.
+
+* [Serialization](content/Serialization/Serialization.q.md) ** - minutes**
+    > Serialization in PHP is used for storing and passing values while retaining their type and structure. In this lesson, you will learn how to use `serialize()` and `unserialize()` functions in PHP.
+
+* [Error Handling](content/ErrorHandling/ErrorHandling.q.md) ** - minutes**
+    > Error handling refers to the process of catching errors detected in the program that requires proper action. Find out how you can handle errors using different types of error handling functions in PHP.
+    
+* [Catching Exceptions](content/CatchingExceptions/CatchingExceptions.q.md) ** - minutes**
+    > In this lesson, you will learn how to use the basic PHP `Exception` classes and the `try` and `catch` statements in handling errors.
+ 
+* [Throwing Exceptions](content/ThrowingExceptions/ThrowingExceptions.q.md) ** - minutes**
+    > Discover how you can throw a new exception using the `throw` statement. 
+
+* [Custom Exceptions](content/CustomExceptions/CustomExceptions.q.md) ** - minutes**
+    > A custom exception is a user-defined error handling function that handles errors when called. Discover how you can create custom error handling functions to manage possible errors in the program.
+
+* [Opening and Closing Files](content/OpeningAndClosingFiles/OpeningAndClosingFiles.q.md) ** - minutes**
+    > In this lesson, you will learn how to open and close a file in PHP using basic file handling functions `fopen()` and `fclose()`.
+
+* [Reading and Writing Files](content/ReadingAndWritingFiles/ReadingAndWritingFiles.q.md) ** - minutes**
+    > Discover how you can read and write a file in PHP using the file handling functions `fread()` and `fwrite()`.
+
+* [Templates](content/Templates/Templates.q.md) ** - minutes**
```

Comment:
Are we going to use a Templating engine for this? Or are we going to roll our own?

Reply:
I think we have to create our own for now since we are trying to introduce only the basics on how to create a template in PHP. Thoughts? Thanks.

Reply:
It might be simpler to use something existing but only cover the basics

Reply:
I think this is possible T and this is noted. Thanks.